trestle/core/resolver/_import.py

```diff
@@ -81,6 +88,19 @@ def __init__(
                     f'Back matter resource resolution needed for profile import failed with error: {str(e)}'
                 )
 
+        uri_type = cache.FetcherFactory.get_uri_type(self._import.href)
+        # if this looks like a relative path to remote source, append parent path
+        if uri_type == cache.FetcherFactory.UriType.LOCAL_FILE and self._parent_url_root:
+            self._import.href = self._parent_url_root + self._import.href
+        # if href is now a remote path, capture its parent path for possible use with child imports that are relative
+        if uri_type not in [cache.FetcherFactory.UriType.LOCAL_FILE, cache.FetcherFactory.UriType.TRESTLE]:
+            head_path, _ = os.path.split(self._import.href)
+            if head_path and head_path[-1] != '/':
+                head_path += '/'
+            self._parent_url_root = head_path
+            logger.info('parent head path %s', head_path)
+        logger.info('import href is %s', self._import.href)
```

Comment:
No need to inform user on it, put it to debug level

Reply:
thanks - I had meant for them to be debug